src/components/Footer/Footer.js

```diff
@@ -1,9 +1,69 @@
-import "./Footer.css";
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
 
+import SearchIcon from '../../assets/search-blue.svg';
+/* import WatchlistIcon from '../../assets/watchlist-add.svg';
+ */
 export default function Footer() {
   return (
-    <div className="footer">
-      <h2>This is the footer</h2>
-    </div>
+    <StyledFooter className="footer">
```

Comment:
Why use `className` when using styled-components?

Reply:
Old code, that was missed while deleting